Jared merge overleaf RE changes into github (#304)
symbiotic-engineering/MDOcean: pubs/renewable-energy-mdo/sections/formulation.tex

Author: ```suggestion
    Priority 2 violations yield an inaccurate but valid objective and constraints.
    This is undesirable during the search and a sign, if active at the solution, that the optimum lies outside the model's accurate range.
```

\fi

\Cref{remdo:tab:requirements} lists each requirement with its priority and enforcement mechanism.

> \citet{mccabe_development_2026} develops the physical and modeling basis for each requirement, such as the underlying geometry, dynamics, hydrostatics, structural, and economic relations, is developed in \citet{mccabe_development_2026}.

Author: ```suggestion
\citet{mccabe_development_2026} develops the physical and modeling basis for each requirement, such as the underlying geometry, dynamics, hydrostatics, structural, and economic relations.
```

    and a maximum-LCOE constraint for the epsilon-constraint procedure of \Cref{remdo:sec:multi-obj-process} (\reqref{LCOE}). 
    \reqref{base_static} dominates \reqref{float_spar_draft}, so the latter need not be implemented independently.
\fi
In summary, the requirements table clearly articulates and prioritizes the necessary items based on both design needs and model assumptions,
and will inform the optimization problem formulation.
%%%%%%%%%%%%%%%%%%%%%%%%
\subsubsection{Design Variables, Parameters, and Constraints}
The encoding of a physical design space into optimization design variables is non-unique.
\ifdefined\DISSERTATION

>     For example, bulk dimensions hvm could be encoded directly as dimensions $T_{f,2}$ and $T_s$, or nondimensionally as dimension $T_s$ and ratio $T_{f,2}/T_s$.

Author: ```suggestion
    For example, the float spar and drafts could be encoded directly as dimensions $T_{f,2}$ and $T_s$, or nondimensionally as dimension $T_s$ and ratio $T_{f,2}/T_s$.
```

>     An encoding may be chosen because it more directly controls the objective, makes the problem convex in the obj or constraints, or helps maintain feasibility and convergence during optimization. The last is pursued here.

Author: ```suggestion
    An encoding may be chosen because it more directly controls the objective, makes the problem convex in the objective or constraints, or helps maintain feasibility and convergence during optimization.
    The last is pursued here.
```

    It is therefore desirable to use only parameters, bounds, and linear constraints (no nonlinear constraints) for the five priority 1 requirements, ensuring they hold even at intermediate points.
    
    Continuing the draft example, two priority 1 requirements, \reqref{float_spar_draft} and \reqref{spar_seafloor_overflow}, contain $T_{f,2}$, while $T_s$ appears in \reqref{float_spar_draft} and \reqref{float_seafloor_overflow}.
    Assuming the other variables involved (e.g.\ $T_{f,1}$, $h$) are parameters or design variables, the dimensional encoding makes all three requirements achievable via linear constraint, while the nondimensional encoding makes \reqref{float_spar_draft} a bound (preferable) but forces \reqref{spar_seafloor_overflow} into a nonlinear constraint (undesired).
    Early results with nondimensional encodings \citep{mccabe_multidisciplinary_2022} showed the optimizer frequently violating nonlinear constraints and causing simulation failure, so the dimensional encoding is selected.
\else

>     For example, bulk dimensions hvm may be encoded directly as dimensions or as nondimensional ratios.

Author: ```suggestion
    For example, the float and spar drafts may be encoded directly as dimensions or as a dimension and a ratio.
```

>     An encoding may be chosen to control the objective directly, induce convexity in the obj or constraints, or maintain feasibility. The last is pursued here, since each requirement can be enforced, in increasing order of optimizer difficulty, via parameters, bounds, linear constraints, or nonlinear constraints, and it is desirable to keep all five priority 1 requirements free of nonlinear constraints so they hold even at intermediate points.

Author: ```suggestion
    An encoding may be chosen to control the objective directly, induce convexity in the objective or constraints, or maintain feasibility.
    The last is pursued here, since each requirement can be enforced, in increasing order of optimizer difficulty, via parameters, bounds, linear constraints, or nonlinear constraints,
    and it is desirable to keep all five priority 1 requirements free of nonlinear constraints so they hold even at intermediate points.
```

>     Applied to the draft example, the dimensional encoding renders the relevant priority 1 requirements achievable by linear constraint whereas the nondimensional encoding forces a nonlinear constraint. Early results with nondimensional encodings \cite{mccabe_multidisciplinary_2022} showed frequent constraint violation and simulation failure, motivating the dimensional choice.

Author: ```suggestion
    Applied to the draft example, the dimensional encoding renders the relevant priority 1 requirements achievable by linear constraint whereas the nondimensional encoding forces a nonlinear constraint.
    Early results with nondimensional encodings \citep{mccabe_multidisciplinary_2022} showed frequent constraint violation and simulation failure, motivating the dimensional choice.
```

    While impedance matching can and should be used to design the actual PTO hardware, this choice adds simplicity and illustrative value at the system level: it highlights the novel non-PTO (e.g.\ structural) aspects of WEC co-design introduced here while still capturing the effect of PTO rating on system performance and maintaining the generality to incorporate PTO optimization later.
    Developing a more detailed PTO cost model that scales with these characteristics would be a necessary next step toward PTO co-design.
\else

>     While the design variables include PTO sizing (force and power ratings), other PTO characteristics such as generator and drivetrain impedances are kept as parameters rather than co-designed. This choice adds simplicity and isolates the novel non-PTO (e.g.\ structural) aspects of co-design introduced here, while still capturing the effect of PTO rating on system performance. 

Author: ```suggestion
    While the design variables include PTO sizing (force and power ratings), other PTO characteristics such as generator and drivetrain impedances are kept as parameters rather than co-designed.
    This choice adds simplicity and isolates the novel non-PTO (e.g.\ structural) aspects of co-design introduced here, while still capturing the effect of PTO rating on system performance. 
```
